packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -357,7 +356,7 @@ class _DismissibleState extends State<Dismissible> with TickerProviderStateMixin
   }
 
   double get _overallDragAxisExtent {
-    final Size size = context.size!;
+    final Size size = context.size;
```

Comment:
This change is done by my IDE, I didn't notice it

Reply:
`BuildContext.size` is a nullable value:

https://github.com/flutter/flutter/blob/2e78ed0ff95ae21c20a98266b0eefcb3aaf762b8/packages/flutter/lib/src/widgets/framework.dart#L2335

It makes sense to revert these changes—there must be something funky going on in your IDE.

Reply:
> `BuildContext.size` is a nullable value:
> 
> https://github.com/flutter/flutter/blob/2e78ed0ff95ae21c20a98266b0eefcb3aaf762b8/packages/flutter/lib/src/widgets/framework.dart#L2335
> 
> It makes sense to revert these changes—there must be something funky going on in your IDE.

Solved, but maybe we should not use `!` in a separate PR?